index.html

```diff
@@ -301,7 +322,7 @@ <h3>VerifiableCredentialSchema2023</h3>
                 "type": ["VerifiableCredential", "VerifiableCredentialSchema2023"],
                 "issuer": "https://example.com/issuers/14",
                 "issuanceDate": "2010-01-01T19:23:24Z",
-                "credentialSubject": {
+                "jsonSchema": {
```

Comment:
-1 to this approach.

The `credentialSubject` property is required in the VCDM, so removing results in an object that isn't a VC.

As an alternative, I suggest taking the same approach as the one taken in  [StatusList2021Credential](https://w3c.github.io/vc-status-list-2021/#statuslist2021credential). That is, make `credentialSubject.type` mandatory. An example of my suggestion below. This would make the vc graph much more palatable, and I believe this uses the more typical JSON-LD patterns. 

```json
{
  "@context": [
      "https://www.w3.org/ns/credentials/v2",
      "https://www.w3.org/ns/credentials/examples/v2"
  ],
  "id": "https://example.com/credentials/3734",
  "type": ["VerifiableCredential", "VerifiableCredentialSchema2023"],
  "issuer": "https://example.com/issuers/14",
  "issuanceDate": "2010-01-01T19:23:24Z",
  "credentialSubject": {
    "id": "https://example.com/credentials/schema/12",
    "type": "JsonSchema2023",
    "jsonSchema": {
      "$id": "https://example.com/schemas/email-credential-schema.json",
      "$schema": "https://json-schema.org/draft/2020-12/schema",
      "name": "EmailCredential",
      "description": "EmailCredential using VerifiableCredentialSchema2023",
      "type": "object",
      "properties": {
        "credentialSubject": {
          "type": "object",
          "properties": {
            "emailAddress": {
              "type": "string",
              "format": "email"
            }
          },
          "required": ["emailAddress"]
        }
      }
    }
  }
}
```

Reply:
Yes, to what @andresuribe87 said above. That's a better way to do it.

Reply:
great suggestion! thank you